imgui-SFML.cpp

```diff
@@ -726,48 +726,33 @@ void Image(const sf::Sprite& sprite, const sf::Vector2f& size, const sf::Color&
 
 /////////////// Image Button Overloads for sf::Texture
 
-bool ImageButton(const sf::Texture& texture, const int framePadding, const sf::Color& bgColor,
-                 const sf::Color& tintColor) {
-    return ImageButton(texture, sf::Vector2f(texture.getSize()), framePadding, bgColor, tintColor);
-}
-
-bool ImageButton(const sf::Texture& texture, const sf::Vector2f& size, const int framePadding,
-                 const sf::Color& bgColor, const sf::Color& tintColor) {
+IMGUI_SFML_API bool ImageButton(const char* id, const sf::Texture& texture,
+                                const sf::Vector2f& size, const sf::Color& bgColor,
+                                const sf::Color& tintColor) {
     ImTextureID textureID = convertGLTextureHandleToImTextureID(texture.getNativeHandle());
 
-    return ImGui::ImageButton(textureID, ImVec2(size.x, size.y), ImVec2(0, 0), ImVec2(1, 1),
-                              framePadding, toImColor(bgColor), toImColor(tintColor));
+    return ImGui::ImageButton(id, textureID, ImVec2(size.x, size.y), ImVec2(0, 0), ImVec2(1, 1),
+                              toImColor(bgColor), toImColor(tintColor));
 }
 
 /////////////// Image Button Overloads for sf::RenderTexture
 
-bool ImageButton(const sf::RenderTexture& texture, const int framePadding, const sf::Color& bgColor,
-                 const sf::Color& tintColor) {
-    return ImageButton(texture, sf::Vector2f(texture.getSize()), framePadding, bgColor, tintColor);
-}
-
-bool ImageButton(const sf::RenderTexture& texture, const sf::Vector2f& size, const int framePadding,
-                 const sf::Color& bgColor, const sf::Color& tintColor) {
+IMGUI_SFML_API bool ImageButton(const char* id, const sf::RenderTexture& texture,
```

Comment:
```suggestion
bool ImageButton(const char* id, const sf::RenderTexture& texture,
```
I don't think you need these macros in the .cpp file. You only need them in the header.

Reply:
You're right, just fixed it